students/olszewski_bartosz/lesson_01_python_basics/apple_sharing.py

```diff
@@ -0,0 +1,6 @@
+print('Podaj liczbę uczniów')
+a = (input())
```

Comment:
Please use additional argument in input for user prompt instead of additional print command:
input('Podaj liczbę uczniów: ')
Then It's handled differently by Snakify (not interpreted as an output)
At best please use English language.